Source/Core/VideoCommon/BoundingBox.cpp

```diff
@@ -51,7 +51,8 @@ void BoundingBox::Flush()
     for (u32 i = start; i < end; ++i)
       m_dirty[i] = false;
 
-    Write(start, std::vector<BBoxType>(m_values.begin() + start, m_values.begin() + end));
+    ERROR_LOG_FMT(VIDEO, "YESSSS");
```

Comment:
Unintentional?

Reply:
Yeah, accidentally committed it. Leftover from testing cases where it gets hit.